lexer/lexer.go

```diff
@@ -55,27 +56,27 @@ func (l *Lexer) NextToken() GojoToken {
 			tok = l.NewToken(TokenOperators["/"], string(l.ch))
 		}
 	case '=':
```

Comment:
can you inline all these operators like so:
```go
case '+', '-' ...
```

Reply:
Fixed in latest commit